src/components/DopplerPlus/ExclusiveForm.js

```diff
@@ -97,6 +128,10 @@ const ExclusiveForm = ({ dependencies: { dopplerLegacyClient, appSessionRef } })
     }
   };
 
+  const handleChange = () => {
```

Comment:
Maybe this function could have a name more specific as handleOthersCheckChange or something like that (I'm not good choosing names 😅 )

Reply:
changed